video/clip.go

```diff
@@ -4,8 +4,20 @@ import (
 	"fmt"
 	"github.com/grafov/m3u8"
 	"github.com/livepeer/catalyst-api/log"
+	ffmpeg "github.com/u2takey/ffmpeg-go"
+	"time"
 )
 
+// format time in secs to be copatible with ffmpeg's expected time syntax
+func formatTime(seconds float64) string {
```

Comment:
This works, but Go does have pretty good builtin time formatting

Reply:
Updated to use the builtin methods.